core/whistle_services-1.0.0/src/bookkeepers/wh_bookkeeper_braintree.erl

```diff
@@ -14,10 +14,15 @@
 -export([commit_transactions/2]).
 -export([charge_transactions/2]).
 -export([already_charged/2]).
+-export([timestamp_to_braintree/1]).
 
-
+-include_lib("braintree/include/braintree.hrl").
 -include("../whistle_services.hrl").
 
+-define(TOPUP_CODE, 3006).
```

Comment:
I agree; I just wasn't sure how pervasive the hard-coded values are in the code.